src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -514,7 +522,7 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
   }
 
   /** Should fail if a value is already set.  The keys in the Map are locally qualified names. */
-  override def setOutputs(workflowId: WorkflowId, key: OutputKey, callOutputs: WorkflowOutputs): Future[Unit] = {
+  override def setOutputs(workflowId: WorkflowId, key: OutputKey, callOutputs: WorkflowOutputs, topLevelOutputs: Map[FullyQualifiedName, WdlType]): Future[Unit] = {
```

Comment:
This is confusing to me, there is a parameter called `callOutputs` which has type `WorkflowOutputs`, and then we have `topLevelOutputs` which are used as a comparator to determine if an output is a top level output.